l10n/en/firefox/features/translate.ftl

```diff
@@ -9,18 +9,18 @@ features-translate-translate-a-webpage-with-firefox = Translate a webpage with {
 features-translate-translate-the-web = Translate the web
 
 # HTML page description
-features-translate-firefox-translations-is-a-built-in = { -brand-name-firefox } Translations is a built-in translation feature that allows you to easily browse the web in your preferred language. Learn more about how this feature in { -brand-name-firefox } works, and how { -brand-name-mozilla } helps keep what you translate private.
+features-translate-firefox-translations-is-a-built-in = { -brand-name-firefox-translations } is a built-in translation feature that allows you to easily browse the web in your preferred language. Learn more about how this feature in { -brand-name-firefox } works, and how { -brand-name-mozilla } helps keep what you translate private.
 features-translate-one-of-the-best-things-about = One of the best things about the internet is that we can access content worldwide. Whether it’s news articles, blogs, or even a review of your latest tech gadget, you can find it all on the seemingly never-ending web. With { -brand-name-firefox }’s latest translation feature, this tool will continuously translate a webpage in real-time.
-features-translate-while-other-browsers-rely-on = While other browsers rely on cloud services, the { -brand-name-firefox } Translations language models are downloaded on the user’s browser and translations are done locally, so { -brand-name-mozilla } doesn’t record what webpages you translate.
+features-translate-while-other-browsers-rely-on = While other browsers rely on cloud services, the { -brand-name-firefox-translations } language models are downloaded on the user’s browser and translations are done locally, so { -brand-name-mozilla } doesn’t record what webpages you translate.
 features-translate-when-you-translate-a-webpage = When you translate a webpage, it stays private
 features-translate-when-your-translations-are = When your translations are processed locally, no data from your chosen device leaves your device or relies on cloud services for translation. This means that { -brand-name-mozilla } doesn’t know what web page you translate, and makes our translation feature stand out in comparison to other translation tools.
 features-translate-what-languages-are-currently = What languages are currently supported?
 
 # This is followed by a localized list of supported languages
-features-translate-the-languages-below-are-what = The languages below are currently supported by the { -brand-name-firefox } Translations feature:
+features-translate-the-languages-below-are-what = The languages below are currently supported by the { -brand-name-firefox-translations } feature:
```

Comment:
I'm afraid this can't be changed without versioning the string — obsoleting the old, adding it as a fallback (thus changing the IDs in the actual template, too) — to make sure it updates in Pontoon properly, see: https://bedrock.readthedocs.io/en/latest/l10n.html#ftl-string-ids (and also the following § Obsolete strings and § Fallback chapters on that page…)

@peiying2 should confirm whether that's the case, but it usually is:/